.. change::
    :tags: bug, mypy
    :tickets: 7496

    Fixed mypy regression where the release of mypy 0.930 added additional
    internal checks to the format of "named types", requiring that they be
    fully qualified and locatable. This broke the mypy plugin for SQLAlchemy,
    raising an assertion error, as there was use of symbols such as
    ``__builtins__`` and other un-locatable or unqualified names that
    previously had not raised any assertions.

